website/docs/r/lb_probe.html.markdown

```diff
@@ -51,11 +51,11 @@ resource "azurerm_lb_probe" "example" {
 The following arguments are supported:
 
 * `name` - (Required) Specifies the name of the Probe.
-* `resource_group_name` - (Required) The name of the resource group in which to create the resource.
 * `loadbalancer_id` - (Required) The ID of the LoadBalancer in which to create the NAT Rule.
 * `protocol` - (Optional) Specifies the protocol of the end point. Possible values are `Http`, `Https` or `Tcp`. If Tcp is specified, a received ACK is required for the probe to be successful. If Http is specified, a 200 OK response from the specified URI is required for the probe to be successful.
 * `port` - (Required) Port on which the Probe queries the backend endpoint. Possible values range from 1 to 65535, inclusive.
 * `request_path` - (Optional) The URI used for requesting health status from the backend endpoint. Required if protocol is set to `Http` or `Https`. Otherwise, it is not allowed.
+* `resource_group_name` - (Optional / **Deprecated**) The name of the resource group in which to create the resource.
```

Comment:
which means this is no longer forcenew

Reply:
Here is using [SchemaResourceGroupNameDeprecated() ](https://github.com/terraform-providers/terraform-provider-azurerm/blob/326ae0569f803b4fbaee7d3a40aaa56ee6568ce8/azurerm/helpers/azure/resource_group.go#L21)which doesn't include forcenew.

Reply:
Here is using [SchemaResourceGroupNameDeprecated() ](https://github.com/terraform-providers/terraform-provider-azurerm/blob/326ae0569f803b4fbaee7d3a40aaa56ee6568ce8/azurerm/helpers/azure/resource_group.go#L21)which doesn't include forcenew.